cooltools/api/dotfinder.py

```diff
@@ -44,16 +61,18 @@ def recommend_kernel_params(binsize):
 
     Returns
     -------
-    (w,p) : (integer, integer)
-        tuple of the outer and inner kernel sizes
+    kernels : {str:ndarray}
+        dictionary of convolution kernels as ndarrays, with their
+        names as keys.
     """
-    # kernel parameters depend on the cooler resolution
-    # TODO: rename w, p to wid, pix probably, or _w, _p to avoid naming conflicts
+    # define "donut" parameters w and p based on the resolution of the data
+    # w is the outside width, and p is the internal width.
     if binsize > 28000:
         # > 30 kb - is probably too much ...
         raise ValueError(
-            f"Provided cooler has resolution {binsize} bases,"
-            " which is too coarse for analysis."
+            f"Provided cooler has resolution of {binsize} bases,"
+            " which is too coarse for analysis. Provide custom kernels"
```

Comment:
too coarse for automated analysis

or 
too coarse for automated kernel recommendation